tests/support/foo-resolve/Bar.tsx

```diff
@@ -0,0 +1,8 @@
+export default class Bar {
```

Comment:
probably `foo-resolve-tsx` would be a more accurate name - it turns out that the interesting part of this test case isn't that we use tsx or that we are letting typescript resolve the files, but rather both are happening at the same time.

I'll rename.